bitnami/concourse/README.md

```diff
@@ -71,22 +71,22 @@ The command removes all the Kubernetes components associated with the chart and
-| `secrets.workerAdditionalCerts` | Additional certificates to add to the worker nodes                                                                                     | `""`                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Name                            | Description                                                                                                                            | Value                 |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| `image.registry`                | image registry                                                                                                                         | `docker.io`           |
+| `image.repository`              | image repository                                                                                                                       | `bitnami/concourse`   |
+| `image.tag`                     | image tag (immutable tags are recommended)                                                                                             | `7.3.2-debian-10-r23` |
+| `image.pullPolicy`              | image pull policy                                                                                                                      | `IfNotPresent`        |
+| `image.pullSecrets`             | image pull secrets                                                                                                                     | `[]`                  |
+| `secrets.localAuth.enabled`     | the use of local authentication (basic auth).                                                                                          | `true`                |
+| `secrets.teamAuthorizedKeys`    | Array of team names and public keys for team external workers. A single                                                                | `[]`                  |
+| `secrets.localUsers`            | List of `username:password` or `username:bcrypted_password` combinations for all your local concourse users. Auto-generated if not set | `""`                  |
+| `secrets.hostKey`               | Concourse Host Keys.                                                                                                                   | `""`                  |
+| `secrets.hostKeyPub`            | Concourse Host Keys.                                                                                                                   | `""`                  |
```

Comment:
We are working on a new parameter for the readme-generator tool (that is the one creating this table automatically) to use "Check values.yaml". In the meantime we need to use an empty string to at least indicate the type of the parameter, otherwise it will show very long strings that can break the table format making the Readme unreadable